core/micro-services/workflow-core/src/main/scala/edu/uci/ics/amber/util/ResourceUtils.scala

```diff
@@ -0,0 +1,17 @@
+package edu.uci.ics.amber.util
```

Comment:
either put it under core, or under DAO. this does not belong here. To avoid unsynced changes, you can leave a TODO here and track it with an issue for future PRs to work on this.

Reply:
https://github.com/Texera/texera/pull/2991 see this PR first. Once it get merged, I will change here